rpm/0004-Introduce-mount-sd-service-that-is-executed-as-user.patch

```diff
@@ -93,9 +93,9 @@ index 00000000..04a745d2
 +++ b/tools/udisksctl-user
 @@ -0,0 +1,4 @@
 +#!/bin/sh
-+ARGS=$@
-+DEVICEUSER=$(loginctl list-sessions | grep seat0 | tr -s " " | cut -d " " -f 4)
-+/bin/su -l $DEVICEUSER -c "/usr/bin/udisksctl $ARGS"
++ARGS="$@"
++DEVICEUSER="$(loginctl list-sessions | fgrep seat0 | tr -s ' ' | cut -d ' ' -f 4)"
++/bin/su -l "$DEVICEUSER" -c "/usr/bin/udisksctl $ARGS"
```

Comment:
See first comment.

Reply:
Which I do not comprehend.  BTW, proper quoting does make a difference in both cases.